Set error status if http has exception and ok status. (#1143)
getsentry/sentry-dotnet: CHANGELOG.md

## Unreleased

### Fixes

> - Set error status to transaction if http has exception and ok status ([#1143](https://github.com/getsentry/sentry-dotnet/pull/1143))

Reviewer: ```suggestion
- End transaction with error if exception is thrown in pipeline ([#1143](https://github.com/getsentry/sentry-dotnet/pull/1143))
```
